docs/getting-started/glossary.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Glossary
+sidebar_position: 2
+description: >- 
+  A glossary of terms used in the Unlock Protocol community and also
+  the greater web3 ecosystem we belong to.
+---
```

Comment:
Let's add the following:
UDT, Key Manager, Referrer, Paywall (our application)

Reply:
Added these plus the ones from @ccarfi document in notion. Updated a few of the definitions that were already there to match his which I liked.